tests/test_wave_01.py

```diff
@@ -158,13 +158,17 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
+    assert janes_data == {
+        "watchlist": [],
+        "watched": [{
+            "title": MOVIE_TITLE_1,
+            "genre": GENRE_1,
+            "rating": RATING_1
+        }]
+    }
```

Comment:
I think this is a good start for this particular assert, but it can be pretty bulky to assert an entire dictionary. When it comes to testing, a good rule of thumb is to make sure our checks are as complete as possible with as little code as possible. Something like the following would work a little better:

```suggestion
    assert updated_data["watched"][0]["title"] ==  MOVIE_TITLE_1       
    assert updated_data["watched"][0]["genre"] ==  GENRE_1
    assert updated_data["watched"][0]["rating"] ==  RATING_1       
```
The overall structure of what our result data structure should look like is tested within the three asserts as well. 

One other thing to note is that we already know that `updated_data` is `janes_data`. Typically, we want our asserts to make sure that the new data looks as expected rather than the original. It just follows the conventions a little more.